dtmi/azureiot/phone_as_a_device/phoneasadevice-1.json

```diff
@@ -0,0 +1,175 @@
+{
+    "@id": "dtmi:azureiot:Phone_As_A_Device:PhoneAsADevice;1",
+    "@type": "Interface",
+    "contents": [
+        {
+            "@type": "Component",
+            "name": "accelerometer",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Accelerometer;1"
+        },
+        {
+            "@type": "Component",
+            "name": "battery",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Battery;1"
+        },
+        {
+            "@type": "Component",
+            "name": "location",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Location;1"
+        },
+        {
+            "@type": "Component",
+            "name": "magnet",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Magnet;1"
+        },
+        {
+            "@type": "Component",
+            "name": "pressure",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Pressure;1"
+        },
+        {
+            "@type": "Component",
+            "displayName": {
```

Comment:
why this component has displayName? should we add it to the other components above?